subprojects/diagnostics/src/integTest/groovy/org/gradle/api/tasks/diagnostics/DependencyInsightReportTaskIntegrationTest.groovy

```diff
@@ -299,6 +300,7 @@ org:leaf:1.0
 """
     }
 
+    @ToBeFixedForConfigurationCache(because = "StyledTextOutput is not serializable")
```

Comment:
❓ It seems the test fails due to some missing output, how is that related to _StyledTextOutput is not serializable_?

Missing output:
```
Dependency resolution failed because of conflict(s) on the following module(s):
   - org:leaf2 between versions 2.5, 1.5 and 1.0
```

Reply:
Yeah, this output should come from `resolutionErrorRenderer` as a `StyledTextOutput`, that's why I've put such `because`

Reply:
@bamboo Found your comment in Slack

Reply:
FTR: I meant a new issue for this particular case.